topdown/object.go

```diff
@@ -148,7 +148,7 @@ func getObjectKeysParam(arrayOrSet ast.Value) (ast.Set, error) {
 			return nil
 		})
 	default:
-		return nil, builtins.NewOperandTypeErr(2, arrayOrSet, ast.TypeName(types.Object{}), ast.TypeName(types.S), ast.TypeName(types.Array{}))
+		return nil, builtins.NewOperandTypeErr(2, arrayOrSet, ast.TypeName(types.Object{}), ast.TypeName(types.Set{}), ast.TypeName(types.Array{}))
```

Comment:
Looks like in all other places, we've spelled them out: https://github.com/open-policy-agent/opa/search?q=NewOperandTypeErr

Reply:
Yeah, that looks easier on the eyes.. I'll do the same here.